index.html

```diff
@@ -25,9 +25,9 @@
         <div class="top">
           <nav>
             <ul class="nav-bar">
-              <li>Portfolio</li>
-              <li>About</li>
-              <li>Contact</li>
+              <li><a href="portfolio" class="portfolio">Portfolio</a></li>
+              <li><a href="contact" class="contacts">Contact</a></li>
+              <li><a href="about" class="about">About</a></li>
             </ul>
           </nav>
         </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if your header still follows the Figma design template, especially the position of the `Humbarger` button in some mobile screen sizes.

| Figma Design | Your Design |
| ------------------- | ----------------- |
| ![Screenshot from 2022-09-05 22-41-11](https://user-images.githubusercontent.com/83042742/188511030-0e441e5c-c491-43c3-8604-161d77b9112a.png) | ![Screenshot from 2022-09-05 22-38-44](https://user-images.githubusercontent.com/83042742/188511056-77772f42-ce8b-49e4-b14a-766ee32e6d9a.png) |